diff -urN libsrs_alt-0.5.org/Makefile.am libsrs_alt-0.5/Makefile.am
--- libsrs_alt-0.5.org/Makefile.am	2004-10-07 13:10:24.038078493 +0200
+++ libsrs_alt-0.5/Makefile.am	2004-10-07 13:12:26.174913541 +0200
@@ -1,11 +1,11 @@
 SUBDIRS = src
 EXTRA_DIST = MTAs
 
-bin_PROGRAMS = srs
-noinst_PROGRAMS = test
+bin_PROGRAMS = srs-alt
+noinst_PROGRAMS = srs-alt-test
 
-test_SOURCES = test.c
-test_LDFLAGS = src/.libs/libsrs_alt.a $(LIBOBJS)
+srs_alt_test_SOURCES = test.c
+srs_alt_test_LDFLAGS = $(top_builddir)/src/libsrs_alt.la
 
-srs_SOURCES = srs.c
-srs_LDFLAGS = -lsrs_alt -Lsrc/ $(LIBOBJS)
+srs_alt_SOURCES = srs.c
+srs_alt_LDFLAGS = $(top_builddir)/src/libsrs_alt.la
diff -urN libsrs_alt-0.5.org/src/Makefile.am libsrs_alt-0.5/src/Makefile.am
--- libsrs_alt-0.5.org/src/Makefile.am	2004-10-07 13:10:24.010084491 +0200
+++ libsrs_alt-0.5/src/Makefile.am	2004-10-07 13:28:39.568386818 +0200
@@ -1,11 +1,5 @@
-AM_CFLAGS =
-EXTRA_DIST = address.h hash.h srs_alt.h timestamp.h hmac.h sha1.h
-include_HEADERS = srs_alt.h
-
-lib_LIBRARIES = libsrs_alt.a
-libsrs_alt_a_SOURCES = address.c hash.c srs_alt.c timestamp.c hmac.c sha1.c
-# libsrs_alt_a_CFLAGS = $(AM_CFLAGS)
-libsrs_alt_a_LIBADD = $(LIBOBJS)
+noinst_HEADERS = address.h hash.h timestamp.h hmac.h sha1.h
+include_HEADERS = srs_alt.h
 
 lib_LTLIBRARIES = libsrs_alt.la
 libsrs_alt_la_LDFLAGS = -version-info 0:5
