diff -urN libsrs_alt-0.3.org/Makefile.am libsrs_alt-0.3/Makefile.am
--- libsrs_alt-0.3.org/Makefile.am	2004-07-18 17:10:33.955042344 +0200
+++ libsrs_alt-0.3/Makefile.am	2004-07-18 17:15:17.249974968 +0200
@@ -1,10 +1,10 @@
 SUBDIRS = src
 EXTRA_DIST = MTAs
 
-bin_PROGRAMS = test srs
+bin_PROGRAMS = srs-test srs
 
-test_SOURCES = test.c
-test_LDFLAGS = src/.libs/libsrs_alt.a
+srs_test_SOURCES = test.c
+srs_test_LDFLAGS = $(top_builddir)/src/libsrs_alt.la
 
 srs_SOURCES = srs.c
-srs_LDFLAGS = -lsrs_alt -Lsrc/
+srs_LDFLAGS = $(top_builddir)/src/libsrs_alt.la
diff -urN libsrs_alt-0.3.org/src/Makefile.am libsrs_alt-0.3/src/Makefile.am
--- libsrs_alt-0.3.org/src/Makefile.am	2004-07-18 17:10:33.892051920 +0200
+++ libsrs_alt-0.3/src/Makefile.am	2004-07-18 17:18:06.699214768 +0200
@@ -1,10 +1,5 @@
-AM_CFLAGS = 
-EXTRA_DIST = address.h hash.h srs_alt.h timestamp.h
-
-lib_LIBRARIES = libsrs_alt.a
-libsrs_alt_a_SOURCES = address.c hash.c srs_alt.c timestamp.c
-libsrs_alt_a_CFLAGS = $(AM_CFLAGS)
-libsrs_alt_a_LIBADD = $(LIBOBJS)
+noinst_HEADERS = address.h hash.h timestamp.h
+include_HEADERS = srs_alt.h
 
 lib_LTLIBRARIES = libsrs_alt.la
 libsrs_alt_la_LDFLAGS = -version-info 0:3 -lcrypto
